packages/core/src/components/document-component/document-component.scss

```diff
@@ -117,6 +134,13 @@
   }
 }
 
+:host([mode="table-of-contents"]) {
+  .editaction-label,
+  .heading {
+    margin-inline-start: document-component.$indent-table-of-contents;
```

Comment:
Doen we hier verstandig aan? Hoe kom je hieraan?

Met margin's positioneert het kind zicht ten opzichte van de parent. Kan dit niet beter door het parent element met padding worden gedaan?

@hansgrimm Wat denk jij hierover?

Reply:
Ik heb padding-inline start op `heading-container` toegepast.

Reply:
Ooohhh wacht sorry ik las weer niet goed 🫨 

Ik was onder de indruk dat deze styling op `:host` werd gezet: Ofwel, het component positioneert zich ten opzichte van zijn (onbekende!) parent.

Maar de styling wordt gewoon op een element gezet wat bij dit component hoort. En een van de tools die we daarvoor kunnen gebruiken is natuurlijk margin!

Sorry sorry sorry. Als je die andere liever hebt is dat helemaal goed 😬